### PR TITLE
Human Pose Annotation

### PR DESCRIPTION
**Pull request by frndmg**
- Select labels for every point upfront
- Drop all human pose keypoints with one control
- Select the area in which you want to drop the points

### CODE REVIEW

File: cvat-ui/src/components/annotation-page/standard-workspace/controls-side-bar/template-control.tsx [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+                                    <Select
+                                        value={item.label.id}
+                                        onChange={handleChangeLabel(item.id)}
+                                    >
+                                        {labels.map((label: any) => (
+                                            <Select.Option
+                                                key={label.id}
+                                                value={label.id}
+                                            >
+                                                {label.name}
```
Comment (MultifokalHirn) [inline cvat-ui/src/components/annotation-page/standard-workspace/controls-side-bar/template-control.tsx:204]: where does the name come from?

**Review by MultifokalHirn — COMMENTED** [review]
RKNEE Schwarzenegger

could you add a way to move all points at once? and what about "rescaling", as in if the person comes close, just make the imagined rectangle around the points larger? and maybe instead of displaying "RKNEE 10" just "RKNEE" and show the rectangle and that has the label PERSON 10?

maybe have the color by group as default?
it interpolates the movement right? didnt see the result in the video if you press play

File: cvat-ui/src/components/annotation-page/standard-workspace/controls-side-bar/template-control.tsx (hunk 1 above)
Comment (frndmg) [reply cvat-ui/src/components/annotation-page/standard-workspace/controls-side-bar/template-control.tsx:204]: `labels` comes from redux, and every label has at least `id` and `name` properties.

You define the labels at the creation of the task.

**Comment by MultifokalHirn** [comment]
@frndmg whats the state here?

**Comment by felix1m** [comment]
*push*

**Comment by frndmg** [comment]
It should be working now

**Comment by MultifokalHirn** [comment]
@frndmg how about we also create a branch that goes from dev and contains these changes and do a pull request to upstream?

**Review by felix1m — APPROVED** [review]
lgtm
What is the behaviour when you just want to annotate bboxes with this version?
Will there be a lot of visual clutter from these changes?

**Comment by frndmg** [comment]
Not at all. This work was made over the new datumaro interface and the changes do not interrupt on normal bbox annotation process.

File: cvat/apps/tracking/tracker.py [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+    for frame in range(start_frame, stop_frame):
+        image_path = task.get_frame_path(frame)
+        img = cv2.imread(image_path)
+        yield frame, img
+
+class RectangleTracker:
+    trackerTypes = ['BOOSTING', 'MIL', 'KCF', 'CSRT', 'MEDIANFLOW', 'TLD',
+        'MOSSE', 'GOTRUN']
+
+    def __init__(self, trackerType = "BOOSTING"):
```
Comment (ahmedtalbi) [inline cvat/apps/tracking/tracker.py:72]: Can you change the default to CSRT or KCF, both are better than BOOSTING.

File: components/tracking/install_opencv_goturn.sh [hunk 3]
```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+cd "$(dirname "$0")"
+wget 'https://github.com/opencv/opencv_extra/raw/c4219d5eb3105ed8e634278fad312a1a8d2c182d/testdata/tracking/goturn.prototxt'
+wget 'https://github.com/opencv/opencv_extra/raw/c4219d5eb3105ed8e634278fad312a1a8d2c182d/testdata/tracking/goturn.caffemodel.zip.001'
+wget 'https://github.com/opencv/opencv_extra/raw/c4219d5eb3105ed8e634278fad312a1a8d2c182d/testdata/tracking/goturn.caffemodel.zip.002'
+wget 'https://github.com/opencv/opencv_extra/raw/c4219d5eb3105ed8e634278fad312a1a8d2c182d/testdata/tracking/goturn.caffemodel.zip.003'
+wget 'https://github.com/opencv/opencv_extra/raw/c4219d5eb3105ed8e634278fad312a1a8d2c182d/testdata/tracking/goturn.caffemodel.zip.004'
```
Comment (ahmedtalbi) [inline components/tracking/install_opencv_goturn.sh]: I would skip Goturn, It is the only one based on deep learning but Honestly the performances are not way better than the correlation filters one, e.g. csrt.

Comment (frndmg) [reply components/tracking/install_opencv_goturn.sh]: This is only a remaining from the base PR. It could be easily removed because we don't use it

**Comment by MultifokalHirn** [comment]
@frndmg please let me know when to merge this, lgtm

**Review by MultifokalHirn — APPROVED** [review]

**Comment by frndmg** [comment]
@MultifokalHirn you can merge it :+1:

**Pull request merged**